us.zoom.Zoom.json

```diff
@@ -11,7 +11,10 @@
         "--socket=pulseaudio",
         "--share=network",
         "--device=all",
-        "--filesystem=home"
+        "--filesystem=home:ro",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-config/zoomus.conf:create",
+        "--filesystem=~/.zoom:create"
```

Comment:
Won't this result in the `zoom` directory being created in the (real) user's `$HOME`?